ui/templates/includes/members_panel.html

```diff
@@ -0,0 +1,41 @@
+<!-- Members Panel -->
+<div class="cd-panel-members from-right" 
+	data-members-url="{% url 'repo-members' repo.slug %}">
+  <header class="cd-panel-header">
+    <h1>Members for Repository: {{ repo.name }}</h1>
+    <a href="#0" class="cd-panel-close">Close</a>
+  </header>
+  <div class="cd-panel-container">
+    <div class="cd-panel-content">
+    	<div id="members-alert"></div>
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-sm-7">
+					<div class="form-group">
+						<div class="input-group">
+					      	<input type="text" name="members-username"
+					      	  placeholder="kerberos id" class="form-control">
+					      	<div class="input-group-addon">@mit.edu</div>
```

Comment:
I would remove this, and maybe do a placeholder instead of `beaver@mit.edu` or similar, also hardcoded again